src/applications/pre-need/config/form.jsx

```diff
@@ -333,38 +334,72 @@ const formConfig = {
           uiSchema: {
             'ui:description': SupportingFilesDescription,
             application: {
-              preneedAttachments: fileUploadUI('Select files to upload', {
-                buttonText: 'Upload file',
-                addAnotherLabel: 'Upload another file',
-                fileUploadUrl: environment.isProduction()
-                  ? `${environment.API_URL}/v0/preneeds/preneed_attachments`
-                  : `${
-                      environment.API_URL
-                    }/simple_forms_api/v1/simple_forms/submit_supporting_documents`,
-                fileTypes: ['pdf'],
-                maxSize: 15728640,
-                hideLabelText: true,
-                createPayload: !environment.isProduction()
-                  ? createPayload
-                  : file => {
-                      const payload = new FormData();
-                      payload.append('preneed_attachment[file_data]', file);
+              // MBMS-66403
+              preneedAttachments: environment.isProduction()
+                ? fileUploadUIPlatform('Select files to upload', {
+                    buttonText: 'Upload file',
+                    addAnotherLabel: 'Upload another file',
+                    fileUploadUrl: environment.isProduction()
+                      ? `${environment.API_URL}/v0/preneeds/preneed_attachments`
+                      : `${
+                          environment.API_URL
+                        }/simple_forms_api/v1/simple_forms/submit_supporting_documents`,
+                    fileTypes: ['pdf'],
+                    maxSize: 15728640,
+                    hideLabelText: true,
+                    createPayload: !environment.isProduction()
+                      ? createPayload
+                      : file => {
+                          const payload = new FormData();
+                          payload.append('preneed_attachment[file_data]', file);
 
-                      return payload;
+                          return payload;
+                        },
+                    parseResponse: !environment.isProduction()
+                      ? parseResponse
+                      : (response, file) => ({
+                          name: file.name,
+                          confirmationCode: response.data.attributes.guid,
```

Comment:
Is this ` confirmationCode: response.data.attributes.guid` correct? Just wanna make sure `.guid` isn't a typo (suppose to be .guide?)

Reply:
This was already in there from before, I just made a copy of the block for the prod flag. The GUID is some kind of identifier for the form after its submitted I think.